tests/post/features/salt_api.feature

```diff
@@ -12,7 +12,10 @@ Feature: SaltAPI
     Scenario: Login to SaltAPI using a ServiceAccount
         Given the Kubernetes API is available
         When we login to SaltAPI with the ServiceAccount 'storage-operator'
-        Then we can invoke '["disk.dump", "state.sls"]' on '*'
+        Then we can ping all minions
```

Comment:
Basically, the fact that this test requires changes shows that there's an issue somewhere in the implementation. This test should remain as-is.

Reply:
Btw this test do not require change but since `storage-opeator` is able to do a `*` on `nodes` he also have this perms.

Basically storage-operator perms in this PR are:
```
        "perms": [
            {
                "*": [
                    "disk.dump", 
                    "state.sls"
                ]
            }, 
            "@jobs", 
            {
                "*": [
                    ".*"
                ]
            }, 
            "@wheel", 
            "@runner", 
            "@jobs"
        ], 
```

Reply:
But it shouldn't have those perms?!? https://github.com/scality/metalk8s/blob/development/2.4/storage-operator/deploy/role.yaml

Reply:
As the cluster role say that he can do `*` on nodes the check we do for groups is approved so we add node-admin groups thats why but if we dont want that the storage operator can do this then we need to either change the check from groups function or changes the rule of his cluster role

Reply:
Oh right, I read the Role too fast. Then that would be a bug I guess, we shouldn't give that many permissions to `storage-operator`.

Reply:
From [this issue](https://github.com/operator-framework/operator-sdk/issues/1226), it seems we should not have kept the default ClusterRole that gets created when the project is initialized with `operator-sdk new`. Since `operator-sdk generate k8s` doesn't generate this file, it would make sense to update it to match our needs and commit it.

Reply:
See #2084

Reply:
Then it's not part of this PR, because we may want to fix it also in 2.4